docs/sources/view-and-analyze-profile-data/profile-cli.md

```diff
@@ -289,7 +289,7 @@ By default, it looks for samples within the last hour, though this can be contro
     - You can provide a custom time range using the `--from` and `--to` flags, for example, `--from="now-3h" --to="now"`.
     - You can specify the profile type via the `--profile-type` flag. The available profile types are listed in the output of the `profilecli query series` command.
     - You can specify the number of leaf locations to keep via the `--keep-locations` flag. The default value is `5`. The Go compiler does not use the full stack trace. Reducing the number helps to minimize the profile size.
-    - You can specify whether the callee aggregation should be used via the `--aggregate-callees` flag. By default, samples are aggregated by the leaf location, ignoring callee line number. Go compiler ignores this information.
+    - You can specify whether the callee aggregation should be used via the `--aggregate-callees` flag. By default, this setting is true, samples are aggregated by the leaf location, ignoring callee line number. Go compiler ignores this information. You can use `--no-aggregate-callees` to disable aggregation. Try both options to see which gives better for your PGO.
```

Comment:
```suggestion
    - You can control whether to use callee aggregation with the `--aggregate-callees` flag. By default, this option is enabled, meaning samples are aggregated based on the leaf location, disregarding the callee line number, which the Go compiler does not utilize. To disable aggregation, use the `--no-aggregate-callees` flag.
```

Reply:
I think that as long as PGO does not rely on the callee line number and we're sure that "Go compiler ignores this information", disabling the aggregation should not have impact the result. Otherwise, this is a bug and we should remove this flag